src/api/api.ts

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+
+export const client = axios.create({
+  baseURL: 'https://mate.academy/students-api',
+})
+
+//#region User
+export const getUserByEmail = (email: string) => {
+  return client.get('/users', {
+    params: {
+      email,
+    }
+  })
+}
```

Comment:
It's a good practice to specify the expected return type for each function. This will make your code more predictable and easier to debug. For example, you could specify that this function returns a Promise that resolves to an AxiosResponse.

Reply:
done